tests/regression/roles/sonic_interfaces/defaults/main.yml

```diff
@@ -196,6 +201,7 @@ tests:
         advertised_speed:
           - 100000
           - 40000
+        unreliable_los: UNRELIABLE_LOS_MODE_ON
```

Comment:
Please move this to test case 17 to make sure it is correctly deleted by this test (due to not being included in the "replacement" configuration for "Interface7").